lightning/src/ln/channelmanager.rs

```diff
@@ -9559,11 +9568,24 @@ where
 							None
 						}
 					},
+				};
+
+				match (responder, response) {
+					(Some(responder), Some(response)) => return responder.respond(response),
+					(None, Some(_)) => {
+						log_trace!(
+							self.logger,
+							"A response was generated, but there is no reply_path specified for sending the response."
+						);
+						return ResponseInstruction::NoResponse
+					},
+					_ => return ResponseInstruction::NoResponse
 				}
 			},
+
 			OffersMessage::InvoiceError(invoice_error) => {
 				log_trace!(self.logger, "Received invoice_error: {}", invoice_error);
-				None
+				return ResponseInstruction::NoResponse
```

Comment:
Add semi-colon

Reply:
Thanks for all the nit suggestions!
Addressed them in [fa60217](https://github.com/lightningdevkit/rust-lightning/commit/fa602175b41920409b20bb85b4fef9f531889f9c)